Fix #44313 `bitnami/suitecrm:7` fails to come up due to group ownership set to `root`

chris001 opened the pull request:
<!--
 Before you open the request please review the following guidelines and tips to help it be more easily integrated:

 - Describe the scope of your change - i.e. what the change does.
 - Describe any known limitations with your change.
 - Please run any tests or examples that can exercise your modified code.

 Thank you for contributing! We will try to test and integrate the change as soon as we can, but be aware we have many GitHub repositories to manage and can't immediately respond to every request. There is no need to bump or check in on a pull request (it will clutter the discussion of the request).

 Also don't be worried if the request is closed or not integrated sometimes the priorities of Bitnami might not match the priorities of the pull request. Don't fret, the open source community thrives on forks and GitHub makes it easy to keep your changes in a forked repo.
 -->

### Description of the change
Fix the group ownership of all files and folders in the web app `bitnami/suitecrm:7`

<!-- Describe the scope of your change - i.e. what the change does. -->

### Benefits
Allows the web app to run.  Without my change, the app failed and exited.
<!-- What benefits will be realized by the code change? -->

### Possible drawbacks
None.

<!-- Describe any known limitations with your change -->

### Applicable issues

<!-- Enter any applicable Issues here (You can reference an issue using #) -->
- fixes #44313

### Additional information

<!-- If there's anything else that's important and relevant to your pull request, mention that information here.-->

chris001 committed:
Fix #44313 `bitnami/suitecrm:7` fails to come up due to group ownership set to `root`.

Signed-off-by: Chris Coleman <colemanchris@hotmail.com>

jotamartos commented on bitnami/suitecrm/7/debian-11/rootfs/opt/bitnami/scripts/suitecrm/postunpack.sh:
```diff
@@ -31,8 +31,8 @@ info "Configuring file permissions for SuiteCRM"
-    # Use daemon:root ownership for compatibility when running as a non-root user
-    configure_permissions_ownership "$dir" -d "775" -f "664" -u "$WEB_SERVER_DAEMON_USER" -g "root"
+    # Use daemon:daemon ownership for compatibility when running as a non-root user
+    configure_permissions_ownership "$dir" -d "775" -f "664" -u "$WEB_SERVER_DAEMON_USER" -g "$WEB_SERVER_DAEMON_USER"
```
The WEB_SERVER_DAEMON_GROUP env var also exists

```suggestion
    configure_permissions_ownership "$dir" -d "775" -f "664" -u "$WEB_SERVER_DAEMON_USER" -g "$WEB_SERVER_DAEMON_GROUP"
```

chris001 replied:
Yes I agree, this makes sense to use this `WEB_SERVER_DAEMON_GROUP` env var!

jotamartos reviewed (changes requested):
Thanks for your contribution. Could you please take a look at my comment?

chris001 committed:
Update bitnami/suitecrm/7/debian-11/rootfs/opt/bitnami/scripts/suitecrm/postunpack.sh

Use `WEB_SERVER_DAEMON_GROUP` for group ownership, instead of `WEB_SERVER_DAEMON_USER`, and instead of `root`.

Co-authored-by: Juan José Martos <jotamartos@gmail.com>
Signed-off-by: Chris Coleman <colemanchris@hotmail.com>

jotamartos commented:
Version `7.13.4-debian-11-r31` includes your changes. Thanks for your contribution.
